Firmware/messages.c

```diff
@@ -16,7 +16,7 @@ const char MSG_BED_DONE[] PROGMEM_I1 = ISTR("Bed done"); ////
 const char MSG_BED_HEATING[] PROGMEM_I1 = ISTR("Bed Heating"); ////
 const char MSG_BED_LEVELING_FAILED_POINT_LOW[] PROGMEM_I1 = ISTR("Bed leveling failed. Sensor didnt trigger. Debris on nozzle? Waiting for reset."); ////c=20 r=5
 const char MSG_BED_SKEW_OFFSET_DETECTION_FITTING_FAILED[] PROGMEM_I1 = ISTR("XYZ calibration failed. Please consult the manual."); ////c=20 r=8
-const char MSG_BELT_STATUS[] PROGMEM_I1 = ISTR("Belt Status");////c=18
+const char MSG_BELT_STATUS[] PROGMEM_I1 = ISTR("Belt status");////c=18
```

Comment:
why changing "S" -> "s"? We have "Bed Heating" and similar menu items

Reply:
Because it was before Belt status with lower case "s". When I added the `MSG_BELT_STATUS` to `messages.cpp/.h` I wrote it like I am used to it with upper "S", but the lang files had still "Belt status" with lower "s" and so have not been translated.

See https://github.com/prusa3d/Prusa-Firmware/commit/377a59c624ba332a82e72c5497d08cbb6776f25a

I am fine to change it to upper case "S" if you like that more.

Reply:
@3d-gussner you are right, lower case "s" fits the Support menu better, other items also have lower case starting letters of second words.